script.js

```diff
@@ -0,0 +1,227 @@
+  },
+
+];
+
+const works = document.getElementById('works');
+
+projects.forEach((project, index) => {
+  const div = document.createElement('div');
+
+  const openModalButtons = document.querySelectorAll('[data-modal-target]');
+  const closeModalButtons = document.querySelectorAll('[data-close-button]');
+  const overlay = document.getElementById('overlay');
+
+  function openModal(modal) {
+    if (modal == null) return;
+    modal.classList.add('active');
+    overlay.classList.add('active');
+  }
+
+  function closeModal(modal) {
+    if (modal == null) return;
+    modal.classList.remove('active');
+    overlay.classList.remove('active');
+  }
+
+  openModalButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const modal = document.querySelector(button.dataset.modalTarget);
+      openModal(modal);
+    });
+  });
+
+  overlay.addEventListener('click', () => {
+    const modals = document.querySelectorAll('.modal.active');
+    modals.forEach((modal) => {
+      closeModal(modal);
+    });
+  });
+
+  closeModalButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const modal = button.closest('.modal');
+      closeModal(modal);
+    });
+  });
+
+  if (index === 0) {
+    div.className = 'card-1';
+    div.innerHTML = `
+    <div class="flex">
+            <h1 class="recent">${project.title}</h1>
+            <hr class="hor" >
+        </div>
+    `;
+  } else if (index === 1) {
+    div.className = 'card-2';
+    div.innerHTML = `
+    <div class="grid">
+      <div class="gymnast"></div>
+      <div class="text-container">
+        <h2 class="recent1">${project.title}</h2>
+        <p class="recentp">${project.text}</p>
+      <div class="languages">
+          <ul>
+            <li><button class="works-btn">CSS</button>
+            <li><button class="works-btn">HTML</button>
+            <li><button class="works-btn">Bootstrap</button>
+            <li><button class="works-btn">Ruby</button>
+            </ul>
+        </div>
+        <button class="see" data-modal-target="#modal" >${project.tags}</button>
+      </div>
+    </div>
+    <div class="grid-projects">
+            <div class="see-proj">
+             <div class="projects-background" id="whitebg"> 
+                <div class="background">
+                <h2 >${project.title}</h2>
+                <p class="para">${project.text}</p>
+                <div class="project-btn">
+                    <ul class="btn-ul">
+                        <li><button class="projects-btn" type="button">HTML</button></li>
+                        <li><button class="projects-btn" type="button">Bootstrap</button></li>
+                        <li><button class="projects-btn" type="button">Ruby</button></li>
+                    </ul>
+                </div>
+            </div>
+            <button data-modal-target="#modal" class="sees-project" id="see-proj-btn">See project</button>
+       </div>
```

Comment:
- Kindly, all cards have to have the `see project` button to view the detail, but you have only on the first card, please update to have it by separating the normal and hover state for all cards 👌
### hover 👇
<img width="367" alt="Screenshot 2023-04-12 at 5 00 27 PM" src="https://user-images.githubusercontent.com/52010789/231482057-26757f79-cff9-453b-b102-c0fe676c2020.png">

### Normal 👇
<img width="367" alt="Screenshot 2023-04-12 at 5 00 59 PM" src="https://user-images.githubusercontent.com/52010789/231482069-4a330a76-e4d6-40d5-8400-089845fbb3cf.png">